File: 백준/Bronze/16727. ICPC/ICPC.c
#include <stdio.h>

int main()
{
    int p1, s1, s2, p2;

    scanf("%d %d\n%d %d", &p1, &s1, &s2, &p2);

    if (p1 + p2 == s1 + s2) {
        if (s1 == p2)
            printf("Penalty");
        else
            printf("%s", (s1 < p2) ? "Persepolis" : "Esteghlal");
    }
    else
        printf("%s", (p1 + p2 > s1 + s2) ? "Persepolis" : "Esteghlal");
}

File: 백준/Bronze/16727. ICPC/ICPC.py
p1, s1 = map(int, input().split())
s2, p2 = map(int, input().split())
if p1 + p2 == s1 + s2:
    if s1 == p2:
        print("Penalty")
    else:
        print("Persepolis" if s1 < p2 else "Esteghlal")
else:
    print("Persepolis" if p1 + p2 > s1 + s2 else "Esteghlal")